packages/components/package.json

```diff
@@ -12,12 +11,21 @@
     "bugs": {
         "url": "https://github.com/frontify/fondue/issues"
     },
-    "main": "src/index.ts",
-    "types": "src/index.ts",
-    "engines": {
-        "node": ">=18"
+    "main": "dist/index.umd.js",
+    "module": "dist/index.es.js",
+    "types": "dist/index.d.ts",
+    "files": [
+        "dist",
+        "CHANGELOG.md"
+    ],
+    "exports": {
+        ".": {
+            "import": "./dist/index.es.js"
+        },
+        "./style.css": "./dist/style.css"
     },
     "scripts": {
+        "build": "vite build",
```

Comment:
This forces us to build everytime we make a change, for better DX, what if we do it only on CI (changing the `main` from `src/index.ts` to `dist/index.js`, ...)?

Reply:
There's a few workaround to improve the DX (I think first we need to identify which is the sandbox/development platform, if we have an `fondue-ui/storybook` then we won't have that problem at all).

To avoid creating separate processes for local and CI, we can easily disable declarations and have `vite build watch` when developing.
I want to keep this draft as a base for discussion, but before we can commit to any solution we need to tackle a few higher level decisions:

- how do we deal with Tailwind
- how do we phase out components
- how do we compose Storybook

Based on these discussions we can review the best strategy to build/ship `fondue-ui`